dapp/module/message/message_test.go

```diff
@@ -3,15 +3,49 @@ package message
 import (
 	"crypto/rand"
 	"encoding/hex"
+	"os"
+	"path/filepath"
 	"testing"
 	"time"
 
 	db "github.com/Bit-Nation/panthalassa/db"
+	km "github.com/Bit-Nation/panthalassa/keyManager"
+	ks "github.com/Bit-Nation/panthalassa/keyStore"
+	mnemonic "github.com/Bit-Nation/panthalassa/mnemonic"
+	storm "github.com/asdine/storm"
 	otto "github.com/robertkrimen/otto"
 	require "github.com/stretchr/testify/require"
 	ed25519 "golang.org/x/crypto/ed25519"
 )
 
+func createStorm() *storm.DB {
+	dbPath, err := filepath.Abs(os.TempDir() + "/" + time.Now().String())
```

Comment:
`time.Now().String()` creates a filename containing spaces and other potentially problematic characters which cause issues and make windows builds panic, we should switch to `.UnixNano()` or similar

>dbPath, err := filepath.Abs(os.TempDir() + "/" + fmt.Sprint(time.Now().UnixNano()))